docs/book/helpers/doctype.md

```diff
@@ -26,39 +26,58 @@ You can also specify a custom doctype as long as it is well-formed.
 The `Doctype` helper is a concrete implementation of the
 [Placeholder helper](placeholder.md).
 
-## Basic Usage
+## Configuration and Basic Usage
 
 You may specify the doctype at any time. However, helpers that depend on the
-doctype for their output will recognize it only after you have set it, so the
-easiest approach is to specify it in your bootstrap:
+doctype for their output will recognize it only after you have set it.
+There are multiple ways to set the doctype:
```

Comment:
This section should be reverted because "Basic Usage" is the first correct entry. The previous version was okay.

Reply:
Done! I borrowed the headline from the `asset` documentation 🙂 

Maybe you should come up with a Documentation guide, like the [Maintainers guide](https://github.com/laminas/technical-steering-committee/blob/main/MAINTAINERS.md) 😉